releasenotes/notes/fix_fips_agent_status-ad9cf20c53bdec9e.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+     Removed the FIPS Proxy status section from the agent status page when running the FIPS Agent. 
```

Comment:
```suggestion
     Remove the FIPS Proxy status section from the Agent status page when running the FIPS Agent. 
```

Reply:
@buraizu QQ: Since these are put into the release notes, would it make more sense for them to be past-tense?

Reply:
@sgnn7 good question, thanks for following up. Release notes are generally kept in present tense to clearly indicate that the change is available in the current version. This is the pattern followed by most of our other release notes as well.

Reply:
May be good to communicate this wider in the Agent group as we're a bit inconsistent on this https://github.com/DataDog/datadog-agent/releases 😅